README.md

```diff
@@ -1,147 +1,63 @@
-# Dependable [![Build Status](https://travis-ci.org/idottv/dependable.png)](https://travis-ci.org/idottv/dependable)
+# Dependable
 
-A minimalist dependency injection framework for node.js.
+A minimalist dependency injection framework for Node.js.
 
-## Example
+## Installation
+To install the latest version:
 
-### Create a container
-
-Create a new container by calling `dependable.container`:
-
-```js
-var dependable = require('dependable'),
-    container = dependable.container();
+```bash
+npm install --save dependable
 ```
 
-### Register some dependencies
+## Usage
 
-Register a few dependencies for later use (a string and an object):
+Create a new container by calling `dependable.container`:
 
-```js
-container.register('occupation', 'tax attorney');
-container.register('transport', {
-  type: 'station wagon',
-  material: 'wood-paneled'
-});
+```JavaScript
+const dependable = require('dependable');
+const container = dependable.container();
 ```
 
-### Register a dependency that has other dependencies
-
-When the argument is a function, the function's arguments are automatically
-populated with the correct dependencies, and the return value of the function
-is registered as the dependency:
-
-```js
-container.register('song', function (occupation, transport, legalStatus) {
-  var song = {};
+To register a new dependency, call `container.register`:
```

Comment:
This can go in another PR, but I think we should go into a _lot_ more detail around what a "dependency" is: dependable only cares about the _return value_ of this function, etc. etc. etc.

Reply:
Agreed. The main point of the README rewrite was to get rid of a lot of verbiage about writing a song in favor of a more concise example. The README as a whole needs some work.